### ISSUE TITLE
VCS status ignores global excludes file

### ISSUE THREAD

**Issue by wqferr** [romkatv/powerlevel10k#460]
I have a line containing `.venv/` in my global gitignore, and git status correctly ignores it. However, the prompt still shows `?1` referring to it.

When I add `.venv/` to the local excludes file (`.git/info/excludes`) or to `.gitignore`, the prompt properly ignores it.

**Comment by wqferr** [comment]
Also, sorry for lazy writing, I'm reporting this while at work. If needed, I can come back and elaborate.

**Comment by romkatv** [comment]
Cannot reproduce. Here's what I've tried:

```zsh
docker run -e TERM -e COLORTERM -it --rm debian:buster bash -uec '
  apt update
  apt install -y git zsh
  mkdir -p ~/.config/git
  echo '.venv/' >~/.config/git/ignore
  mkdir -p ~/repo/.venv
  touch ~/repo/.venv/foo
  git -C ~/repo init
  git clone --depth=1 https://github.com/romkatv/powerlevel10k.git ~/powerlevel10k
  cat >~/.zshrc <<\END
POWERLEVEL9K_DISABLE_CONFIGURATION_WIZARD=true
source ~/powerlevel10k/powerlevel10k.zsh-theme
END
  cd ~/repo
  exec zsh'
```

Prompt shows clean repo.

> If needed, I can come back and elaborate.

Please do.

**Comment by wqferr** [comment]
I'll try to provide screenshots before going to work. If I can't, I'll be able to in about 11~12h from now.

Thanks for the response.

**Comment by romkatv** [comment]
Screenshots would be nice. Instructions how to reproduce would be even better. Otherwise I'll have to guess them from screenshots.

**Comment by wqferr** [comment]
Yes, I'll include the commands and outputs to reproduce in the screenshots.

As far as I could tell, instructions to reproduce are really just what you did there. I'm going to test it on my setup and do the same thing as you did, showing the output.

**Comment by wqferr** [comment]
Well, apparently my personal notebook is no longer... Booting...

I'll install it on my work computer as well (was already planning on doing this) and see if I can reproduce.

**Comment by wqferr** [comment]
Welp, can't reproduce it at work. It seems something else was causing it in my home setup. Considering today's events, I'd say it was not the only thing not properly setup...

Sorry for wasting your time :/

**Comment by romkatv** [comment]
No worries. Please reopen if you see something similar again.

**Comment by wqferr** [comment]
Will do!

**Comment by gschmitz22** [comment]
tldr: if you see this, restart your terminal

I had this issue today after adding a global .gitignore. git status was showing clean but the prompt was showing ?1. Once I closed my terminal and reopened though, it was correct.

**Comment by romkatv** [comment]
@gschmitz22 Thanks for the info!

**Comment by wqferr** [comment]
Interesting, I thought I had done that. But good to know someone got a solution!
